platform/admin/modules/admin-modules-docs/src/main/java/org/codice/ui/admin/docs/DocsSparkApplication.java

```diff
@@ -44,7 +43,7 @@ public void init() {
 
   public File getDocumentationHtml() {
 
-    if (!Files.exists(Paths.get(DOCS_ROOT_DIR))) {
+    if (!Paths.get(DOCS_ROOT_DIR).toFile().exists()) {
```

Comment:
❓ Why not keep the Path when defining the static attribute above instead of continuously re-creating one here?